contracts/interfaces/IPriceFeedOracle.sol

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: GPL-3.0-only
 
 pragma solidity >=0.5.0;
+pragma experimental ABIEncoderV2;
```

Comment:
This was bugging me but just now I realized it's actually not needed. Please remove.